README.md

```diff
@@ -60,6 +60,38 @@ Now run Go launcher and navigate to "http://localhost:8000/quick-launch?url=" pa
 e.g."http://localhost:8000/quick-launch?url=http://localhost:7777/1_0001.json"
 ```
 
+### Deployment with [Helm](https://helm.sh/)
+
+To deploy this application with helm, you must have a kubernetes cluster already running and be logged into the cluster.
+
+Log in to the cluster using:
+```
+gcloud container clusters get-credentials survey-runner --region <region> --project <gcp_project_id>
+```
+
+You need to have Helm installed locally
+
+1. Install Helm with `brew install kubernetes-helm` and then run `helm init --client-only`
+
+1. Install Helm Tiller plugin for tillerless deploys `helm plugin install https://github.com/rimusz/helm-tiller`
+
+---
+
+The following environment variables must be set when deploying the app.
+- RUNNER_URL
+- DOCKER_REGISTRY
+- IMAGE_TAG
+
+To deploy to a cluster you can run the following command
+
+```
+./k8s/deploy_app.sh
+```
+##### Example
+ ```
+RUNNER_URL=https://example.com ./k8s/deploy_app.sh
```

Comment:
This example can go now that three variables are required and are documented above.